Nighttime-Operations/Auxiliary-Telescope/AT-Shutdown/Shutdown.rst

```diff
@@ -1,19 +1,164 @@
+
+**1. Closing the M1 cover.**
+: First, disable ATAOS corrections. Then, close the M1 cover and vent gates.  
+If mirror is opened, ``shutdown`` script will not able to close the dome slit. 
+
+**2. Closing the Dome Shutter.**
+: Close the dome shutter if it is opened. 
+If the dropout door is open, you can close manually via button at the AuxTel or via notebook with the command below.
+
+.. code-block:: yaml
+
+   await atcs.rem.atdome.cmd_closeShutter.set_start() 
+    
+**3. Parking the Dome** 
+: The parking position is az=285.0 deg. Then, the dome following is disabled after parking. 
+
+**4. Parking the Telescope**
+: The parking position is El = 80 deg, Az = 0.0 deg, rot = 0.0 deg for telescope.
+Tracking will then be stopped.  
+
+**5. Putting CSCs in Standby.**
+: The AT CSC components ``ATMCS``, ``ATOS``, ``ATPtg``, ``ATDome``, ``ATDomeTrajector``, ``ATOS``, ``ATPneumatics``, ``ATHexapod``  will be change to Standby. 
+
+After parking and shutdown for Auxillary Telescope are completed, you can check the status of Dome and Telescope on the LOVE. 
+
+   
+Taking Morning Calibrations
+---------------------------
+
+After ``shutdown.py`` script is completed, check CSC Summary on LOVE.
+Check the CSCS for LATISS are in enable status, then run ``make_latiss_calibration.py`` with the following configuration.
+See more details on :ref:`LATISS-Combined-Calibrations-Procedure-LATISS-Combined-Calibrations-Generation-Procedure`.
+
+.. code-block:: yaml
+
+    script_mode: BIAS_DARK
+    generate_calibrations: False
+
+
+If it is not working, 
+
+1. Run ``make_image_latiss.py`` to take biases with the following configurations:
+
+.. code-block:: yaml
+
+    image_type: BIAS
+    nimages: 50
+    reason: 'dailyCals' 
+
+2. Run ``make_image_latiss.py`` to take dark images with the following configurations:
+
+.. code-block:: yaml
+    
+    image_type: DARK
+    nimages: 23
+    exp_times: [5, 5, 5, 5, 5, 15, 15, 30, 30, 30, 30, 30, 30, 30, 30, 30, 30, 30, 30, 30, 30, 30, 30]
+    reason: "dailyCals"
+
+
+*Standby_latiss*
+----------------
+
+After parking the AuxTel and taking daytime calibration, the LATISS CSCs are still in ENABLED state.
+Run ``auxtel/standby_latss.py`` on ATQueue to set all CSC for LATISS (``ATCamera``, ``ATOODS``, ``ATHeaderService``, ``ATSpectrograph``) 
+ 
+
+
+Ending the Night
+----------------
+
+During the shutting down procedures, you can check and complete all logging and administrative tasks including:
+
+- Finish filling out the nightlog, including time loss, weather loss, and gathering fault reports for the night.
+- Send the summary and night log link to rubinobs-nightlog mailing list (rubin-night-log [at] lists.lsst.org). 
+- Leave the summary of the night on Slack Channels - #rubin_daytime_tasks and #summit-auxtel 
+
+Closing the Vent Gate
+---------------------
+
+This step can be proceed any points of this shutdown steps, but it is recommeneded doing on the way down to the hotel at the end of the night.  
```

Comment:
typo: any point (singular)
re frase: ...it is recommended to do it on the way down...